sui/open_rpc/spec/openrpc.json

```diff
@@ -0,0 +1,2087 @@
+{
```

Comment:
I was hoping that we can generate ts typing from this schema(so that we can automate the syncing) and tried loading this file with https://github.com/open-rpc/typings, unfortunately I got an out of memory error

```
npx open-rpc-typings --output-ts .

<--- Last few GCs --->

[45334:0x160008000]    40122 ms: Mark-sweep 4042.4 (4128.9) -> 4031.2 (4133.9) MB, 4860.8 / 0.1 ms  (average mu = 0.185, current mu = 0.054) allocation failure scavenge might not succeed
[45334:0x160008000]    45056 ms: Mark-sweep 4047.5 (4133.9) -> 4034.2 (4136.7) MB, 4902.3 / 0.0 ms  (average mu = 0.098, current mu = 0.006) allocation failure scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0x102c0a104 node::Abort() [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 2: 0x102c0b104 node::OnFatalError(char const*, char const*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 3: 0x102d6243c v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 4: 0x102d623d0 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 5: 0x102eb21d0 v8::internal::Heap::EnsureFillerObjectAtTop() [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 6: 0x102eb43a0 v8::internal::Heap::ComputeMutatorUtilization(char const*, double, double) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 7: 0x102eb2a04 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 8: 0x102eb0c74 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
 9: 0x102ebdbbc v8::internal::Heap::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
10: 0x102ebdc3c v8::internal::Heap::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
11: 0x102e956d4 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
12: 0x10314a96c v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
13: 0x102a7d5ec Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/opt/homebrew/Cellar/node/17.5.0/bin/node]
14: 0x1093cbd50
15: 0x1093c9fe8
16: 0x10959b3e4
17: 0x1095b9c1c
18: 0x10935e304
19: 0x102ac56f8 Builtins_PromiseFulfillReactionJob [/opt/homebrew/Cellar/node/17.5.0/bin/node]
20: 0x102a32c54 Builtins_RunMicrotasks [/opt/homebrew/Cellar/node/17.5.0/bin/node]
21: 0x102a0fe84 Builtins_JSRunMicrotasksEntry [/opt/homebrew/Cellar/node/17.5.0/bin/node]
22: 0x102e4e668 v8::internal::(anonymous namespace)::Invoke(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
23: 0x102e4e9d4 v8::internal::(anonymous namespace)::InvokeWithTryCatch(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
24: 0x102e6faf4 v8::internal::MicrotaskQueue::RunMicrotasks(v8::internal::Isolate*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
25: 0x102e6f920 v8::internal::MicrotaskQueue::PerformCheckpointInternal(v8::Isolate*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
26: 0x102b41ab4 node::InternalCallbackScope::Close() [/opt/homebrew/Cellar/node/17.5.0/bin/node]
27: 0x102b423fc node::InternalMakeCallback(node::Environment*, v8::Local<v8::Object>, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*, node::async_context) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
28: 0x102b5cc9c node::AsyncWrap::MakeCallback(v8::Local<v8::Function>, int, v8::Local<v8::Value>*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
29: 0x102c0f620 node::fs::FSReqCallback::Resolve(v8::Local<v8::Value>) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
30: 0x102c1094c node::fs::AfterNoArgs(uv_fs_s*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
31: 0x104f338c0 uv__work_done [/opt/homebrew/Cellar/libuv/1.43.0/lib/libuv.1.dylib]
32: 0x104f36c38 uv__async_io [/opt/homebrew/Cellar/libuv/1.43.0/lib/libuv.1.dylib]
33: 0x104f46458 uv__io_poll [/opt/homebrew/Cellar/libuv/1.43.0/lib/libuv.1.dylib]
34: 0x104f37058 uv_run [/opt/homebrew/Cellar/libuv/1.43.0/lib/libuv.1.dylib]
35: 0x102b42c60 node::SpinEventLoop(node::Environment*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
36: 0x102c47b58 node::NodeMainInstance::Run(int*, node::Environment*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
37: 0x102c477ac node::NodeMainInstance::Run(node::EnvSerializeInfo const*) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
38: 0x102bd7dc0 node::Start(int, char**) [/opt/homebrew/Cellar/node/17.5.0/bin/node]
39: 0x1051cd0f4
[1]    45333 abort      npx open-rpc-typings --output-ts .
```

Reply:
hmmm the schema works with https://playground.open-rpc.org so it should be valid... I will investigate, we have some cyclic structs in the response type `TypeTag` maybe that's causing problem...

I will work on simplifying the responses after I finish other devnet tasks, hopefully we can remove TypeTag and make the schema simpler